scripts/exaqm_make_ics.sh

```diff
@@ -602,15 +32,16 @@ fi
 #
 #-----------------------------------------------------------------------
 #
-mv out.atm.tile${TILE_RGNL}.nc \
-        ${INPUT_DATA}/${NET}.${cycle}${dot_ensmem}.gfs_data.tile${TILE_RGNL}.halo${NH0}.nc
+#
+mv out.atm.tile7.nc \
+        ${COMIN}/${cyc}/${NET}.${cycle}.gfs_data.tile7.halo0.nc
 
-mv out.sfc.tile${TILE_RGNL}.nc \
-        ${INPUT_DATA}/${NET}.${cycle}${dot_ensmem}.sfc_data.tile${TILE_RGNL}.halo${NH0}.nc
+mv out.sfc.tile7.nc \
+        ${COMIN}/${cyc}/${NET}.${cycle}.sfc_data.tile7.halo0.nc
 
-mv gfs_ctrl.nc ${INPUT_DATA}/${NET}.${cycle}${dot_ensmem}.gfs_ctrl.nc
+mv gfs_ctrl.nc ${COMIN}/${cyc}/${NET}.${cycle}.gfs_ctrl.nc
 
-mv gfs.bndy.nc ${INPUT_DATA}/${NET}.${cycle}${dot_ensmem}.gfs_bndy.tile${TILE_RGNL}.f000.nc
```

Comment:
Why use SRW-defined variables when it's more straightforward to us the NCO-standard ones?


Also, `tile7` and `halo0` aren't actually configurable in Production AQM now. Why use variables for them?